recipe/menu-v2.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "https://schemas.conda.io/menuinst-1.schema.json",
+    "menu_name": "Anaconda ({{ DISTRIBUTION_NAME }})",
+    "menu_items": [
+        {
+            "name": {
+                "target_environment_is_base": "Spyder",
+                "target_environment_is_not_base": "Spyder ({{ ENV_NAME }})"
+            },
+            "description": "Scientific PYthon Development EnviRonment",
+            "activate": false,
+            "terminal": false,
+            "icon": "{{ MENU_DIR }}/spyder.{{ ICON_EXT }}",
+            "command": ["{{ PREFIX }}/bin/spyder", "$@"],
+            "platforms": {
+                "win": {
+                    "desktop": false,
+                    "quicklaunch": false,
+                    "app_user_model_id": "anaconda.Spyder",
+                    "command": ["{{ PYTHONW }}", "{{ SCRIPTS_DIR }}/spyder-script.py"]
```

Comment:
menu 1 has an additional shortcut for reset.
https://github.com/AnacondaRecipes/spyder-feedstock/blob/menuinst2/recipe/menu-v1.json#L13

Reply:
Is that something we want to keep?

Reply:
My guess is that, since the reset icon hasn't gotten any updated branding, this isn't really something even spyder is interested in: https://github.com/spyder-ide/spyder/tree/master/img_src